src/components/sheet/sheet-action-buttons/SheetActionButton.tsx

```diff
@@ -115,8 +143,9 @@ const SheetActionButton = ({
       scaleTo={disabled ? 1 : scaleTo}
       size={size}
       testID={`${testID}-action-button`}
-      {...props}
+      marginBottom={marginBottom}
```

Comment:
Margin? No need for fall-through props?

Reply:
That's my mistake, this should be left since there are JS components using this which may be using the passthrough props